core-bundle/contao/elements/ContentAlias.php

```diff
@@ -36,7 +36,7 @@ public function generate()
 		System::getContainer()->get('contao.cache.entity_tags')->tagWithModelInstance($objElement);
 
 		// Clone the model, so we do not modify the shared model in the registry
-		$objModel = $objElement->cloneOriginal();
+		$objModel = $objElement->cloneDetached();
```

Comment:
```suggestion
		$objModel = $objElement->cloneOriginal();
		$objModel->preventSaving(false);
```

In case we don‘t want a new method, we could also call `preventSaving()` here. But I‘m fine either way.

Reply:
That's not the same thing, because `cloneOriginal()` duplicates the database values and drops everything that has been changed on the model!